tasks/4/index.js

```diff
@@ -1,7 +1,5 @@
 // 4. From the array of numbers to deduce a new array of values of each element multiplying by 2
 
-const getMultiplyingArray = (array) => {
-  // your code
-};
+const getMultiplyingArray = (array) => array.map(number => number * 2);
```

Comment:
`.map((number) => number * 2);`
Please always write parameters of function in parentheses